octoprint_discordremote/command.py

```diff
@@ -94,9 +94,10 @@ def timelapse(self):
     def help(self):
         builder = EmbedBuilder()
         builder.set_title('Commands, Parameters and Description')
+        # builder.set_description("All commands should be prefaced with the bot prefix: '%s'" % self.plugin.get_settings().get(["prefix"]))
```

Comment:
@OctopusProteins Did you mean to leave this uncommented?

Reply:
Yes, it was remnant from when I was going to leave the commands without their prefix on the help list. You can remove it.

Reply:
Okay, is it okay if I leave it to you to remove it? Also, the unittests have been broken, please fix them. I believe you need to properly mock the calls to "self.plugin.get_settings().get(["prefix"])"

Something like (In unittests/test_command.py):
    _settings = mock.Mock()
    _settings.get = mock.Mock()
    _settings.get.return_value = "/"

Reply:
Okay, I'll do it, and i'll send the commit your way so you can have a look.

Reply:
Awesome, thanks!